examples/list-binding-pattern-in-match-statement/list_binding_pattern_in_match_statement.out

```diff
@@ -0,0 +1,6 @@
+$ bal run list_binding_pattern_in_match_statement.bal
+List all files
+Remove all files
+Remove files: ["a.bal","b.bal"]
+move: a.bal
```

Comment:
```suggestion
Move the`a.bal` file.
```

Reply:
The output here is generated by [`io:println(command, ": ", file);`](https://github.com/ballerina-platform/ballerina-distribution/pull/3850/files#diff-0333cd163ca5b3ecccd356128f029910b8986b6beaeaaee6cdc831158c95c325R25) which prints whatever the command passed. In this case, the command passed is [`move`](https://github.com/ballerina-platform/ballerina-distribution/pull/3850/files#diff-0333cd163ca5b3ecccd356128f029910b8986b6beaeaaee6cdc831158c95c325R37). The command in [list_binding_pattern_in_match_statement.bal](https://github.com/ballerina-platform/ballerina-distribution/pull/3850/files#diff-0333cd163ca5b3ecccd356128f029910b8986b6beaeaaee6cdc831158c95c325R34-R38) can be capitalized so that it prints `Move` here, but it will be inconsistent with the rest of the commands there.

WDYT @praneesha?